lib/Runtime/Base/FunctionBody.h

```diff
@@ -1862,6 +1862,7 @@ namespace Js
         WriteBarrierPtr<void*> inlineCaches;
         InlineCachePointerArray<PolymorphicInlineCache> polymorphicInlineCaches; // Contains the latest polymorphic inline caches
         WriteBarrierPtr<PropertyId> cacheIdToPropertyIdMap;
+        WriteBarrierPtr<PropertyIdArray> formalsPropIdArray;
```

Comment:
Done.

---
In reply to: [67777247](https://github.com/Microsoft/ChakraCore/pull/1135#discussion_r67777247) [](ancestors = 67777247)